dfw-init.php

```diff
@@ -228,8 +228,11 @@ private function targeting() {
 		}
 
 		// Templates
-		if ( is_single() ) {
+		// change from is-single() to is_singular() to target pages; only define 'inurl' parameter on non-category, and non-home pages
```

Comment:
remove this comment

Reply:
This change request was marked as "resolved" but no changes were made. Please remove the comment before marking this change request as "resolved".